core/services/synchronization/stats_pusher.go

```diff
@@ -73,32 +80,73 @@ func (eq *StatsPusher) Close() error {
 	return eq.WSClient.Close()
 }
 
-func (eq *StatsPusher) pollEvents(parentCtx context.Context) {
+type response struct {
+	Status int `json:"status"`
+}
+
+func (eq *StatsPusher) syncEvent(event *models.SyncEvent) error {
+	eq.WSClient.Send([]byte(event.Body))
```

Comment:
Note: Right now we have an asynchronous `sendPump`, which doesn't fit our new synchronous sending model as well. Perhaps in a future refactor, we can make this synchronous for more immediate and better send error handling.